### ISSUE TITLE
Fails to enable systemd unit on install [Fedora 30]

### ISSUE THREAD

**Issue by iko-deleted** [testcab/ansible-role-code-server#2]
Upon initial install, the systemd unit isn't properly detected.

```

TASK [pallxk.code_server : Enable code-server service] ***************************************************************
fatal: [192.168.1.100]: FAILED! => {"changed": false, "msg": "Could not find the requested service code-server: host"}

```

I initially thought this was something wrong with the script, but I sshed into the server and tried `daemon-reload`+starting the service manually, and it didn't work either. Rebooting the system fixed the problem.

My best guess is that there's a bug in systemd that prevents it from seeing unit files in /usr/local/... unless the folder exists at boot. I'll do some testing and report it upstream when I find the time.

This issue is mostly just to remind myself to do something.

**Comment by cortopy** [comment]
It's been a long time since I used Fedora but it could be that the systemd directory for services is not right. There's a file for RedHat family and I guess it should already be working, but just in case I just did a PR for Debian because I was having similar issues

**Comment by pallxk** [comment]
Fixed via 0cd1b2ee5ee70ff2c456b83bcc3ed6c502b1a169. Thanks for you report!
